src/main/resources/fabric.mod.json

```
@@ -23,7 +23,7 @@
   "depends": {
     "fabricloader": ">=0.14.6",
     "fabric": "*",
-    "minecraft": "1.20.1",
+    "minecraft": ">=1.20",
```

```suggestion
    "minecraft": "1.20.*",
```

i would prefer to play it safe and keep this set to the 1.20.x series for now - just in case there's breaking changes in 1.21. also thanks for catching this!